packages/ui-scripts/src/post-publish.ts

```diff
@@ -64,7 +64,7 @@ async function postPublish(
     createGitTagForRelease(releaseVersion)
 
     if (hasJiraConfig(config)) {
-      issueKeys = await getIssuesInRelease(jiraProjectKey)
```

Comment:
This is not an async function.

Reply:
hmm I'd get rid of this code, we're not connecting to JIRA

Reply:
Done, since the `post-publish` effectively just called this JIRA function I've removed the command altogether.

Reply:
Since we didn't use the `post-publish` script at all I've deleted it altogether.